docs/source/model_design/vivarium_model_components/impairments/index.rst

```diff
@@ -33,17 +33,224 @@ Modeling Impairments
+risk for diabetes and blindness, the issue would be more exacerbated than if 
+different risk factors caused each. 
+
+One example of this can be seen in our IV Iron Simulation and more specifically 
+in modeling :ref:`anemia <2019_anemia_impairment>` and :ref:`maternal hemorrhage <2019_cause_maternal_hemorrhage_incidence>`. In this case, YLDs for maternal hemorrhage 
+were not counted due to overlap with anemia YLDs. The anemia YLDs included both those due to 
+both maternal hemorrhage and other causes. 
+ 
+
+Include All Possible Causes
++++++++++++++++++++++++++++
+
+Another solution might be to include all of the causes that touch blindness, 
+which would therefore include all of the blindness sequelae that make 
+up the impairment. In this approach you would not model "blindness" as an impairment, 
+but could get all of the same data from sequelae. 
+
+For the case of blindness, there are approximately 15 separate causes, not including 
+all neonatal causes. This would mean having 15 cause models included! This is 
+likely to be a lot of work, and would probably not provide much extra information. 
+
+Furthermore, you still have the issue that cause models are independent in Vivarium. 
+Simulants could still get blindness due to multiple causes, leading to the same undercounting 
+of total blindness. 
+
+For practical and logistic reasons, this approach is not recommeneded. 
+
+Include a Hybrid Cause & Impairment Model
++++++++++++++++++++++++++++++++++++++++++
+
+Another option is to include a single cause model that includes both the cause 
+and impairment. One example of this is in the CVD model where IHD and Heart Failure 
+are included in the same :ref:`cause model <2019_cause_ihd>`.
+
+By including both the cause and impairment in a single modeling component, 
+they are no longer independent which solves our prior problems of overlap. 
+In addition, you can define part of the model to overlap - for example states 
+could inlcude diabetes, blindness due to diabetes, and blindness due to other 
+causes. 
+
+Including cause models structured in this way does sometimes lead to strange "predictive" 
+qualities for simulants. For example, a simulant with blindness from other 
+causes might not be able to develop diabetes to avoid overlap with blindess 
+due to diabetes. In real life, these conditions exist separately, but can't in 
+the model. 
+
+This approach should not create bias in your model and generally works well. 
+However, if you want to include many causes that overlap with an impairment 
+(e.g., including diabetes, meningitis, glaucoma, and vitamin A deficiency, all of 
+which cause blindness) 
+using this approach would make an extremely complex single "cause" model that 
+likely wouldn't be feasible to create or V&V. 
+
+Therefore, considering your modeling situation and the requirements is important 
+in choosing your modeling approach. 
+
+Have Causes Act as "Risks"
+++++++++++++++++++++++++++
```

Comment:
Overall, I think this section is pretty speculative -- I think I'd like to add a TODO here noting that we don't yet know exactly how to do this.